cnd/src/storage/db/tables/btc_dai_orders.rs

```diff
@@ -3,7 +3,7 @@ use crate::storage::{
         schema::*,
         wrapper_types::{Erc20Amount, Satoshis, WeiPerSat},
     },
-    NotOpen, Order, Text,
+    NotOpen, NotSettling, Order, Text,
```

Comment:
Any reason why you defined this error outside the only file that uses it?

Reply:
I was thinking about that actually, for now all errors are just inside the db `errors` module but I would actually want them to be co-located yes.

I'll do that in one of my follow-ups!